src/main/java/ua/tc/marketplace/model/dto/ad/AdDto.java

```diff
@@ -20,29 +20,18 @@
  */
 public record AdDto(
     Long id,
-
-    @NotNull(message = "Author ID cannot be null")
-    Long authorId,
-
+    @NotNull(message = "Author ID cannot be null") Long authorId,
     @NotEmpty(message = "Title cannot be empty")
-    @Size(max = 100, message = "Title cannot be longer than 100 characters")
-    String title,
-
-    @NotEmpty(message = "Description cannot be empty")
-    String description,
-
+        @Size(max = 100, message = "Title cannot be longer than 100 characters")
+        String title,
+    @NotEmpty(message = "Description cannot be empty") String description,
     @NotNull(message = "Price cannot be null")
-    @DecimalMin(value = "0.0", inclusive = false, message = "Price must be greater than zero")
-    BigDecimal price,
-
+        @DecimalMin(value = "0.0", message = "Price must be zero or positive")
+        BigDecimal price,
     List<Photo> photos,
```

Comment:
i suggest to use "List<Long> photoIds" here

Reply:
explain why do you think so? `Photo` is just json object, it does not contain files. And do you think it is still redundant?

Reply:
Maybe I'm digging too deep. Even without files, I guess there will be many different situations for sending dto, and not every situation needs to have full info. Especially pathes of the files. That is absolutely redundant info for frontend and PhotoDto should be used. But at first place it should be discussed with front end devs, lets leave it for later. Question is off for now